action.yml

```diff
@@ -42,14 +42,14 @@ runs:
         GH_ACTION_REF: ${{ github.action_ref || 'main' }}
       working-directory: ${{ steps.inputs.outputs.working_directory }}
       run: |
-        wget --output-document=${{ github.workspace}}/.git/ansible-lint-requirements.txt https://raw.githubusercontent.com/ansible/ansible-lint/$GH_ACTION_REF/.config/requirements-lock.txt
+        wget --output-document=${{ steps.inputs.outputs.working_directory }}/.git/ansible-lint-requirements.txt https://raw.githubusercontent.com/ansible/ansible-lint/$GH_ACTION_REF/.config/requirements-lock.txt
```

Comment:
The `${{ steps.inputs.outputs.working_directory }}/` prefix is redundant, as there is already for `working-directory:` setting the working directory for the shell command (`wget` included); I just created #4232 to hopefully fix this.

Reply:
nope, this can not work. When you run this on a container it can not get correct `working-directory` from GITHUB environment variables. GITHUB environment variables are only available for the RUNNER, not on a container in RUNNER.